components/experimental/src/transliterate/transliterator/mod.rs

```diff
@@ -402,11 +418,74 @@ impl Transliterator {
             + ?Sized,
         F: Fn(&Locale) -> Option<Result<Box<dyn CustomTransliterator>, DataError>>,
     {
+        #[cfg(test)]
+        println!("{marker_attributes:?}");
         let req = DataRequest {
             id: DataIdentifierBorrowed::for_marker_attributes(marker_attributes),
-            ..Default::default()
+            metadata: {
+                let mut m = DataRequestMetadata::default();
+                m.silent = true;
+                m
+            },
         };
-        let transliterator = transliterator_provider.load(req)?.payload;
+        let transliterator = if let Some(t) = transliterator_provider
+            .load(req)
+            .allow_identifier_not_found()?
+        {
+            t.payload
+        } else if let Ok(locale) = marker_attributes.parse::<Locale>() {
+            let fallbacker = fallbacker()?;
+            let fallbacker_configed =
+                fallbacker.for_config(TransliteratorRulesV1::INFO.fallback_config);
+
+            let mut transform_extensions = locale.extensions.transform.clone();
+            let mut source_iterator = fallbacker_configed.fallback_for(
+                transform_extensions
+                    .lang
+                    .take()
+                    .unwrap_or(LanguageIdentifier::UNKNOWN)
+                    .into(),
+            );
+            let mut target_iterator = fallbacker_configed.fallback_for(locale.id.clone().into());
+
+            'target: loop {
+                if target_iterator.get().is_unknown() {
+                    Err(DataErrorKind::IdentifierNotFound.with_marker(TransliteratorRulesV1::INFO))?;
+                }
+                'source: loop {
+                    if source_iterator.get().is_unknown() {
+                        break 'source;
+                    }
+                    let mut candidate = target_iterator.get().into_locale();
+                    candidate.extensions.transform = transform_extensions.clone();
+                    candidate.extensions.transform.lang =
+                        Some(source_iterator.get().into_locale().id);
+                    if let Ok(t) = Self::load_rbt(
```

Comment:
Question/Suggestion: why the recursive `load_rbt`? Should we also be checking the user `lookup` function? Or should you be calling directly into `transliterator_provider.load` like before?